### ISSUE TITLE
A new alternative: openit.io

### ISSUE THREAD

**Issue by thangngoc89** [juliangruber/ghub.io#27]
Hey @juliangruber 

Thank you for created such an amazing service. I was a long time user of your service. But recently, ghub.io becomes unresponsive. Redirect takes several seconds and it looks like there is no activities on this repo. So I decided to go ahead and create a new alternative. 

Introducing openit: https://openit.io/
Example link: https://openit.io/js/review , scoped: https://openit/js/@phenomic/core

I focus on aggressive caching so I can talk directly to the CouchDB API instead of synching it locally. This remove a lot of headaches.

And again, thank you for being such a great inspiration

**Comment by juliangruber** [comment]
if it's slow, probably @rvagg only needs to restart it 😁

**Comment by juliangruber** [comment]
nice! looks really good to me!

**Comment by juliangruber** [comment]
if you added `https://openit.io/*` as an alias to `https://openit.io/js*`, I'm pretty sure, @rvagg would also be interested in shutting down ghub.io and only setting up a domain redirect

**Comment by thangngoc89** [comment]
@juliangruber Thank you for your kind words. I'm implementing the alias right now since I'm mainly a JS dev ==> save key strokes. npm eco should have more love then  others ;)

**Comment by thangngoc89** [comment]
Also, I didn't implement the path open yet.

**Comment by thangngoc89** [comment]
@juliangruber Done with alias `https://openit.io/*` to `https://openit.io/js*` 
![preact](https://user-images.githubusercontent.com/3049054/28126257-2d43f618-6753-11e7-9e76-fdf14f9beb89.gif)

**Comment by rvagg** [comment]
yeah, something's up with the process, I restarted it and cleaned out the database and it's back to its speedy self .. for now

**Comment by zeke** [comment]
Hey this looks great! 

> I'm pretty sure, [@rvagg](https://github.com/rvagg) would also be interested in shutting down ghub.io and only setting up a domain redirect

👍

**Comment by charlike** [comment]
Looks looks, but it is down with 522 error.

I'm long ghub user too and i'm a bit tired of it in last few months.

So, Before even looking over the issues here, i just created https://ghub.now.sh - uses `micro` and `now` so it will be extremely fast and stable.

/cc @rvagg @juliangruber

**Comment by juliangruber** [comment]
Awesome, keep us posted!

**Comment by juliangruber** [comment]
Another fix for now would be to simply remove the npm cache and request the package meta data for each http request. It seems to be this cache that is causing the majority of problems.
